contracts/cw1-whitelist/src/integration_tests.rs

```diff
@@ -0,0 +1,129 @@
+use crate::msg::{AdminListResponse, ExecuteMsg, InstantiateMsg, QueryMsg};
+use anyhow::{anyhow, Result};
+use assert_matches::assert_matches;
+use cosmwasm_std::testing::{mock_env, MockApi, MockStorage};
+use cosmwasm_std::{to_binary, Addr, CosmosMsg, Empty, QueryRequest, WasmMsg, WasmQuery};
+use cw1::Cw1Contract;
+use cw_multi_test::{App, AppResponse, BankKeeper, Contract, ContractWrapper, Executor};
+use derivative::Derivative;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref OWNER: String = "owner".to_string();
```

Comment:
Why to involve static var instead of instantiate it in `Suite`?

Reply:
It doesn't actually matter I think.
I did it as you say earlier, but it was bit annoying to have same unused-but-required string instantiated in 3 different places. (unused because it has no use in this case)
But yeah, maybe I've overdone it. I'm looking at this now and `Owner` inside `Suite` looks fine as well.

Reply:
Technically you are right, but basically I find global statics code smell unless they are well argued. Here it is testing code, so it basically doesn't matter, as in test code we often allow for simplifications we wont have in production, but this is not this much a simplification, and also there is nothing wrong with having it in Suite, and additionally - it introduces dependency (a heavy one) which is not such a big deal, but in this case I don't see a reason.

Reply:
Removed.